jit-application/src/test/java/org/dcsa/jit/integration/TimestampRoutingServiceIT.java

```diff
@@ -0,0 +1,270 @@
+              "camel.redelivery-delay=1000") // lower retry delay so the tests run quicker
+          .applyTo(configurableApplicationContext.getEnvironment());
+    }
+  }
+
+  @BeforeEach
+  void init() {
+
+    AddressTO addressTO =
+        AddressTO.builder()
+            .name("John")
+            .street("Kronprincessegade")
+            .streetNumber("54")
+            .floor("5")
+            .postCode("1306")
+            .city("København")
+            .stateRegion("N/A")
+            .country("Denmark")
+            .build();
+
+    timestamp =
+        TimestampTO.builder()
+            .publisher(
+                PartyTO.builder()
+                    .partyName("Asseco Denmark")
+                    .taxReference1("CVR-25645774")
+                    .taxReference2("CVR-25645774")
+                    .publicKey("eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkFzaW")
+                    .address(addressTO)
+                    .identifyingCodes(
+                        List.of(
+                            IdentifyingCodeTO.builder()
+                                .DCSAResponsibleAgencyCode(DCSAResponsibleAgencyCode.SMDG)
+                                .partyCode("EMC")
+                                .codeListName("Evergreen Marine Corporation")
+                                .build()))
+                    .nmftaCode("CSK")
+                    .build())
+            .vessel(TimestampVesselTO.builder().vesselIMONumber("9321483").build())
+            .publisherRole(PublisherRole.CA)
+            .vesselIMONumber("9321483")
+            .UNLocationCode("SGSIN")
+            .facilityTypeCode(FacilityTypeCodeOPR.BRTH)
+            .eventClassifierCode(EventClassifierCode.EST)
+            .operationsEventTypeCode(OperationsEventTypeCode.ARRI)
+            .eventLocation(
+                LocationTO.builder()
+                    .locationName("Eiffel Tower")
+                    .latitude("48.8585500")
+                    .longitude("2.294492036")
+                    .UNLocationCode("SGSIN")
+                    .address(addressTO)
+                    .facilityCode("PSABT")
+                    .facilityCodeListProvider(FacilityCodeListProvider.SMDG)
+                    .build())
+            .vesselPosition(
+                LocationTO.builder().latitude("48.8585500").longitude("2.294492036").build())
+            .portCallPhaseTypeCode(null)
+            .portCallPhaseTypeCode(PortCallPhaseTypeCode.INBD)
+            .eventDateTime(OffsetDateTime.now())
+            .carrierVoyageNumber("2103S")
+            .carrierImportVoyageNumber("2103S")
+            .carrierExportVoyageNumber("2103S")
+            .carrierServiceCode("FE1")
+            .transportCallSequenceNumber(2)
+            .remark("Port closed due to strike")
+            .delayReasonCode("WEA")
+            .build();
+  }
+
+  @Test
+  void after_valid_timestamp_insert_should_notify_successfully_on_valid_credentials_test()
```

Comment:
why use snake case instead of camel case here?

Reply:
Resolved